corehq/apps/importer/tests.py

```diff
@@ -343,7 +386,8 @@ def testLocationOwner(self):
             ['', 'duplicate-location-name', '', duplicate_loc.name],
             ['', 'non-case-owning-name', '', improper_loc.name],
         ])
-        cases = {c.name: c for c in list(get_cases_in_domain(self.domain))}
+        case_ids = self.accessor.get_case_ids_in_domain()
+        cases = {c.name: c for c in list(self.accessor.get_cases(case_ids))}
```

Comment:
Any particular reason, why the accessor API doesn't already have a way to get cases in one call instead of two?